cmd/copy.go

```diff
@@ -1480,16 +1480,28 @@ func init() {
 		Example:    copyCmdExample,
 		Args: func(cmd *cobra.Command, args []string) error {
 			if len(args) == 1 { // redirection
-				if stdinPipeIn, err := isStdinPipeIn(); stdinPipeIn == true {
+				// Enforce the usage of from-to flag when pipes are involved
+				if raw.fromTo == "" {
+					return fmt.Errorf("fatal: from-to argument required")
```

Comment:
>return fmt.Errorf("fatal: from-to argument required") [](start = 5, length = 53)

I'd say to avoid confusion, we should add "when piping" to the end.

Reply:
Sure Adele. Private drop is already out there so we cannot change that but since we are planning to do these changes in the actual flow as well, I'll add "while piping" to the message. 👍